HandsUp/Assets/CardViewScene/Scripts/Category/CategoryManager.cs

```diff
@@ -0,0 +1,180 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using UnityEngine.UI;
+
+public class CategoryManager : MonoBehaviour
+{
+    private PlayerManager playerManager;
+    public GameObject categoryItem;
+
+    private List<Category> categories;
+    private List<CustomCategory> customCategories;
+
+
+    private void Awake()
+    {
+        playerManager = GameObject.Find("PlayerManager").GetComponent<PlayerManager>();
+        categories = new List<Category>();
+        customCategories = new List<CustomCategory>();
+    }
+
+
+    public void InitCategories()
+    {
+        GameObject.Find("Canvas").transform.Find("CategoriesScrollView").gameObject.SetActive(true);
+        GameObject.Find("Canvas").transform.Find("CardsScrollView").gameObject.SetActive(false);
+
+        if (categories.Count <= 0)
+        {
+            GetBuitInCategoriesFromServer();
+        }
+
+        if (customCategories.Count > 0)
+        {
+            GameObject[] content = GameObject.FindGameObjectsWithTag("customCategoryItem");
+            for (int i = 0; i < content.Length; i++)
+            {
+                Destroy(content[i]);
+            }
+            customCategories.Clear();
+        }
+
+        if (playerManager.GetUserId() >= 0)
+        {
+            GetCustomCategoriesFromServer(playerManager.GetUserId());
+        }
+    }
+
+
+    private void GetBuitInCategoriesFromServer()
```

Comment:
이 부분 오타 수정해야할 것 같습니다!
```suggestion
    private void GetBuiltInCategoriesFromServer()
```

Reply:
수정했습니다!